pkgs/by-name/li/libmypaint/gettext-0.25.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/configure.ac b/configure.ac
+index 416d9fe..f941741 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -248,12 +248,13 @@ if test "x$enable_i18n" != "xno"; then
+   AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE",
+                      [The prefix for our gettext translation domains.])
+   AC_SUBST(GETTEXT_PACKAGE)
+-  IT_PROG_INTLTOOL
+-  AM_GLIB_GNU_GETTEXT
++IT_PROG_INTLTOOL
++AM_GNU_GETTEXT([external])
++AM_GNU_GETTEXT_VERSION([0.21])
```

Comment:
Please explain why in the commit message. (`AM_GLIB_GNU_GETTEXT` is obsolete and this is the replacement.)